Sources/TSCBasic/Process.swift

```diff
@@ -350,11 +351,35 @@ public final class Process {
                 pathString: ProcessEnv.path,
                 currentWorkingDirectory: cwdOpt
             )
+#if os(Windows)
+            var searchPaths = Array<AbsolutePath>()
+            var buffer = Array<WCHAR>(repeating: 0, count: Int(MAX_PATH + 1))
+
+            // The 32-bit Windows system directory
+            GetSystemDirectoryW(&buffer, .init(MAX_PATH + 1))
+            searchPaths.append(AbsolutePath(String(decodingCString: buffer, as: UTF16.self)))
+
+            // The 16-bit Windows system directory
+            if let systemDrive = ProcessEnv.vars["systemdrive"],
```

Comment:
Please use `GetSystemDrive` rather than the environment variable.

Reply:
I didn’t see such API. `getenv("systemdrive")` seems to be what is commonly used to retrieve the system drive.

Reply:
Ah, right, sorry about that.  I was thinking of the combination of `GetSystemDirectoryW` and `PathStripToRootW`.

Reply:
I think I was wrong about the 16-bit system directory. It is placed under Windows directory rather than the root. Now the implementation should not involve any environment varaibles.